migrations/20210914160014_group_extensions.js

```diff
@@ -0,0 +1,32 @@
+exports.up = async function (knex) {
+  await knex.schema.createTable('extensions', table => {
+    table.increments().primary()
+    table.text('type')
+    table.timestamp('created_at')
+    table.timestamp('updated_at')
+  })
+
+  await knex.schema.createTable('group_extensions', table => {
+    table.increments().primary()
+    table.bigInteger('group_id').references('id').inTable('groups').notNullable()
+    table.bigInteger('extension_id').references('id').inTable('extensions').notNullable()
+    table.jsonb('data')
+    table.boolean('active').defaultsTo(true)
+    table.index('group_id')
+    table.timestamp('created_at')
+    table.timestamp('updated_at')
+  })
+
+  await knex.schema.alterTable('groups', async (table) => {
+    table.text('type')
```

Comment:
hmm, isn't text a large text field without a size so shouldn't this be a string type?

Reply:
Yeah, I didn't see this distinction. Obviously type fields don't need a big character limit. Is this worth changing?

![image](https://user-images.githubusercontent.com/5193218/156631200-3fe53a5f-856b-476f-aae7-a780604cf62c.png)

Reply:
no performance differnence?? fascinating. ok then I guess its fine

Reply:
so did we decide to leave as is?